app/api/mealsController/meals.controller.js

```diff
@@ -0,0 +1,83 @@
+import mealsDb from "../db/meals.db";
+
+class MealsController {
+  //   get all meals
+  getAllMeals(req, res) {
+    res.status(200).send({
+      success: true,
+      message: "meals retrieved successfully",
+      meals: mealsDb
+    });
+  }
+
+  //   add a meal
+  addMeal(req, res) {
+    if (!req.body.name || !req.body.description || !req.body.price) {
+      return res.status(404).send({
+        success: false,
+        message: "name, description and price required"
+      });
+    }
+
+    const meal = {
+      id: mealsDb.length + 1,
```

Comment:
Refrain from using the length of the array to generate ids. Use something like shortId instead.